spec/support/saml_response_doc.rb

```diff
@@ -0,0 +1,264 @@
+require_relative 'saml_auth_helper'
```

Comment:
not pure, no. Basically gets rid of all the module-only methods in favor of the class.

`require_relative` is a shorthand for 'require in a dir relative to the current file' -- mostly so as not to need to manage the include path, which is done explicitly in `spec/rails_helper.rb` with
```
Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
```

since that does not actually alter the Ruby include path, the require_relative means It Just Works.